server/controllers/profile.js

```diff
@@ -0,0 +1,14 @@
+const getUserAccounts = require('../database/queries/profile');
+
+const profileInfo = (request, response) => {
+	const { userId } = request.body;
+	getUserAccounts(userId)
+		.then((result) => {
+			if (result.rowCount !== 0) {
+				response.send(result.rows);
+			}
+		})
+		.catch((err) => console.log(err));
```

Comment:
instead of console.log, you have to send a response to the front end with an error message

Reply:
Done